modules/Bio/EnsEMBL/CDS.pm

```diff
@@ -241,6 +241,21 @@ sub phase {
 }
 
 
+=head2 feature_so_acc
+
+  Example    : print $cds->feature_so_acc;
+  Description: This method returns a string containing the SO accession number of CDS.
+               Overrides Bio::EnsEMBL::Feature::feature_so_acc
+  Returntype : string (Sequence Ontology accession number)
+
+=cut
+
+sub feature_so_acc {
```

Comment:
This is a lot of copy-paste. 

I would have gone for:
$self->{so_term} = 'SO:0000316';

and in Bio::EnsEMBL::Feature
```
sub ->feature_so_acc {
  my $self = shift;
  if ( ### one of the exceptional classes ) {
    ...
  } else {
    return $self->{so_term};
  }
}
```
Maybe with some handling of undef etc.